src/main/scala/com/gu/newsletterlistcleanse/braze/BrazeClient.scala

```diff
@@ -90,10 +81,10 @@ object BrazeClient {
         .send()
 
 
-      parseValidateResponse[ExportIdBrazeResponse](response).map(response =>
-        response match {
-        case Left(error) => Left(error)
-        case Right(validResponse) => Right(validResponse.invalidUserIds)
+      response.map(parseValidateResponse[ExportIdBrazeResponse]).map(parsedResponse =>
+        parsedResponse match {
+          case Left(error) => Left(error)
+          case Right(validResponse) => Right(validResponse.invalidUserIds)
         }
       )
```

Comment:
I had missed that, you can use syntactic sugar here to make it more concise
```suggestion
      response.map(parseValidateResponse[ExportIdBrazeResponse]).map {
        case Left(error) => Left(error)
        case Right(validResponse) => Right(validResponse.invalidUserIds)
      }
```